pom.xml

```diff
@@ -61,6 +47,9 @@
 			${project.basedir}/target/site/jacoco-it/jacoco.xml
 		</sonar.coverage.jacoco.xmlReportPaths>
 		<sonar.jacoco.reports>${project.basedir}/target/site</sonar.jacoco.reports>
+
+    <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
```

Comment:
indentation

Reply:
Doesn't look like this locally

Reply:
Pretty sure its a github issue, you could clone down and check to be 100% I guess

Reply:
Wait I do see its completely messed up if I check source code in repo. I wonder why it doesn tlook like this for me locally

Reply:
Hmm if you edit it on here I think you'll be able to correct it